packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -131,12 +258,83 @@ mod tests {
     async fn get_entity_types() -> Result<(), sqlx::Error> {
         let pool = PostgresDatabase::new(&DB_INFO).await?.pool;
 
-        let _rows: Vec<EntityType> = sqlx::query_as("SELECT * from entity_types")
+        let _rows: Vec<EntityTypeRow> = sqlx::query_as("SELECT * from entity_types")
             .fetch_all(&pool)
             .await
             .report()
             .attach_printable("Could not select entity types")?;
 
         Ok(())
     }
+
+    #[ignore]
+    #[tokio::test]
```

Comment:
I think we should wait merging this PR until we can run the tests in CI (https://github.com/hashintel/hash/pull/733).
```suggestion
    #[tokio::test]
    #[cfg_attr(miri, ignore = "miri can't run in async context")]
```